db/seeds.rb

```diff
@@ -6,3 +6,12 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
  User.create(username: 'admin', password: "password")
+
+usernames = ['apple', 'orange', 'banana', 'pear']
```

Comment:
The [faker](https://github.com/faker-ruby/faker) gem is a great resource for seed/test data!

Reply:
Oh yes, that's right! I meant to add that. Good catch & thank you!